### ISSUE TITLE
Explore Dashboard for Passport Stamps

### ISSUE THREAD

**Issue by erichfi** [gitcoinco/passport#1036]
Gitcoin Passport currently aggregates various online identifiers (stamps) from multiple sources to build a user's identity score. To better understand user behavior and adoption, the Passport team wants to build a dashboard that displays the average number of stamps per Passport holder. The team should explore and evaluate different ways to build this dashboard, including using web frameworks like Django or other data visualization tools that may be suitable for this particular use case.

The team should aim to
- [x] Obtain the current average number of stamps per Passport holder, regardless of the approach used to build the dashboard.
- [x] Additionally, the team should suggest the most efficient and effective way to build the dashboard that meets the needs of the project and its users. This could include recommendations on data visualization tools, web frameworks, or any other technologies that may be appropriate.

**Comment by nutrina** [comment]
Here is an idea:
1. export data from the DB to S3 (as CSV files)
2. one could import that CSV files into any analytics tools (like R Studio, Jupyter Notebooks, ...)
3. we might even create a cron job to do the export regularly (like once a day for example)
4. the data from s3 could be imported into Google BigQuery for example and evtl. made public.

The key is having access to the data. This is what the CSV exports (or BigQuery) can enable.
Once the data access is solved, analysing the data becomes less of a problem.

**Comment by lucianHymer** [comment]
Hey @erichfi, see the average in the Results section at the bottom of this page: https://github.com/gitcoinco/passport-scorer-analytics/blob/average-stamps/stamps_per_passport.ipynb

We did what Gerald mentioned above and exported this data to a CSV. We then loaded it into a Jupyter Notebook, which is the file above.

Let me know what you think! If you all like this, we could automate the CSV dump so that it happens daily.
